distribution/ddf-common/src/main/resources/etc/certs/CertNew.sh

```diff
@@ -1,72 +1,52 @@
-#! /bin/bash
-
-if [ -z "$OPENSSL" ]; then OPENSSL=openssl; fi
-if [ -z "$DAYS" ] ; then DAYS="-days 365" ; fi	# 1 year
-if [ -z "$CATOP" ] ; then CATOP=./demoCA ; fi
-
-CACERT=./cacert.pem
-
-REQ="$OPENSSL req"
-CA="$OPENSSL ca"
-PKCS12="openssl pkcs12"
-
-export OPENSSL_CONF=openssl-demo.cnf
-if [[ ! -w ${CATOP}/index.txt ]]; then
-  touch ${CATOP}/index.txt
-fi
-if [[ ! -w ${CATOP}/serial ]]; then
-  echo 01 > ${CATOP}/serial
-fi
-
-read -p "Enter server or user name (common name): " CN
-if [[ -d ${CN} ]]; then
-  echo "${CN} already has been generated. Use a different name or clean up ${CN}/
-    and openssl's index and serial lists in ${CATOP}/ first."
-  exit 1
+#! /bin/bash 
```

Comment:
I know we didn't have this in the ticket, but we also need a .bat script for exposing this cert generator. Can you make another ticket to do that work (unless you think you can tack it in here).